internal/host/transition.go

```diff
@@ -436,6 +454,15 @@ func (th *transitionHandler) HasInstallationInProgressTimedOut(sw stateswitch.St
 	return time.Since(time.Time(sHost.host.Progress.StageUpdatedAt)) > maxDuration, nil
 }
 
+func IsInstallationPaused(clusterID strfmt.UUID, db *gorm.DB) (bool, error) {
+	var cluster common.Cluster
+	err := db.Select("status").Take(&cluster, "id = ?", clusterID.String()).Error
+	if err != nil {
+		return false, err
+	}
+	return swag.StringValue(cluster.Status) == models.ClusterStatusPendingForInput, nil
```

Comment:
the cluster is not pending for input - it should be a new state, please take a look at the comments in the PR.